### DISCUSSION TITLE
how do add address module

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#17065] [How to] [answered]
in the sales order module, there is an address module 
<img width="624" alt="image" src="https://user-images.githubusercontent.com/18006681/236613426-1456c55e-6691-4bdb-af66-47252876d7b6.png">
I want to add the address module in the vendor enquiries module too.
<img width="626" alt="image" src="https://user-images.githubusercontent.com/18006681/236613452-9b0c5705-f884-487a-aeca-4854f92e9366.png">

please help advise how to do, thanks in advance

**Comment by apcloic** [comment]
Hi,

Address is not a module so you can simply create an extra block and address fields in the Vendor Enquiries module.

Regarding field's name, I suggest you to follow the same logic (addresslevel1a, addresslevel2a, addresslevel5a, etc) like in contact/account/leads, etc. That way, it will be easier to map address fields from another module if needed.

Extra function like "copy address from" need custom code, you may look at source code to see how it works and clone the function in another module like Vendor Enquiries.

Regards,

**Reply by newminder** [reply]
okay, thanks very much. I thought there is an easy way to solve this.  let me try.

**Comment by newminder** [comment, marked as answer]
<img width="437" alt="image" src="https://github.com/YetiForceCompany/YetiForceCRM/assets/18006681/8cefcf85-19bc-4782-a7eb-98fdb1ce262b">
solved.
